src/main/java/com/qualitestgroup/java/assessment/framework/pages/DemoPage.java

```diff
@@ -16,6 +16,18 @@
     public CommonPage page;
     public GlobalVars global;
 
+    @FindBy(xpath = "//div[@class='category-cards']/div")
+    private WebElement elementCard;
+
+    @FindBy(xpath = "(//ul[@class='menu-list']/li[2])")
```

Comment:
At the moment, there aren't other ways for us to fetch the elements apart from using xpath. There are no id's and using text is as fragile as xpath.

Reply:
I dont agree. This xpath can be improved. It is too unstable to have an xpath return multiple elements when we still have additional ways to narrow the search. This would be a better xpath: "//ul[@class='menu-list']/li/span[text()='Check Box']"